chainntnfs/txnotifier.go

```diff
@@ -484,34 +506,62 @@ func NewTxNotifier(startHeight uint32, reorgSafetyLimit uint32,
 	}
 }
 
+// validateConfRegistration validates all of the parameters required to
+// successfully register a confirmation notification.
+func (n *TxNotifier) validateConfRegistration(txid *chainhash.Hash,
```

Comment:
Does the function name describe the code inside enough? It indeed validates parameters, but is also creates the ntfn object.

Reply:
Fixed.